SECURITY.md

```diff
@@ -0,0 +1,120 @@
+# Security Policy
```

Comment:
Kind of relevant here, but only tangentially.

Thoughts on simply leaving this file out for multi-geth?

Reply:
Removed!